init.sh

```diff
@@ -133,6 +133,7 @@ if [[ $SYSTEM_GCC -eq 0 ]]; then
   $SOURCE_DIR/source/gcc/build.sh
 
   # Stage one is done, we can upgrade our compiler
+  GCC_INSTALL_PREFIX="$BUILD_DIR/gcc-$GCC_VERSION"
```

Comment:
If this is llvm-specific should it be made public here? Everyone pulls in init.sh. Any reason we can't just put this right in the llvm script?